core/src/test/java/org/testcontainers/containers/ComposeContainerDocTest.java

```diff
@@ -0,0 +1,122 @@
+package org.testcontainers.containers;
+
+import org.junit.Test;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.io.File;
+import java.time.Duration;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ComposeContainerDocTest {
```

Comment:
I think we can use existing examples instead of adding new test just for docs.

Reply:
@eddumelendez - Yes, I also considered this - there are a few classes from which we can derive example, for eg: [ComposeContainerTest](https://github.com/testcontainers/testcontainers-java/blob/main/core/src/test/java/org/testcontainers/junit/ComposeContainerTest.java)

However, if we rely only on the existing tests, we may end up with examples using a mix of different services. For now, I’ve tried to keep a consistent setup using two services throughout the doc page.

On the other hand, I can also enrich some of the existing test classes with a few extra tests and avoid creating an extra test class. For instance, adding 2-3 tests to the class linked above might be enough. What do you think?

Reply:
@eddumelendez - I've pushed some changes now, trying to make as much as possible from the existing tests